code-challenges.test.js

```diff
@@ -52,5 +96,21 @@ const numbersToAdd2 = [0, 7, -8, 12]
 const numbersToAdd3 = []
 // Expected output: []
 
+//create a test statement for the array
+describe("sumOfArray", () => {
+  it("Creates a function that takes in an array and returns an array of the accumulating sum", () => {
+    expect(sumOfArray(numbersToAdd1, numbersToAdd2, numbersToAdd3)).toEqual(result)
+  })
+})
+
+
 
 // b) Create the function that makes the test pass.
+
+//try to make a function that will iterate through the array and return that same set of value
+//use map function
+
+const addingSum = numbersToAdd1.map(value => {
+  return value
+})
+console.log(addingSum(sumOfArray(""))
```

Comment:
You definitely have the right idea to use .map to iterate over the array, however once turned into a function, this will only return the same array.  The prompt is asking to accumulate the sum on each iteration.  A great way to check to see if you are approaching the problem correctly is by looking at your input variables and the expected output that we give you in these problems.  Please make sure to add some more pseudocode on this problem as well!